src/quizzes/chapter-1/1.3.2/vim-specifics-quiz.js

```diff
@@ -0,0 +1,60 @@
+const rawQuizdown = `
+---
+shuffleQuestions: true
+shuffleAnswers: true
+---
+
+# A file has the following contents:<br>
+\`text tex|t text\`<br>
+The '|' signifies where the cursor is at<br>
+Vim is in normal mode<br><br>
+What are the contents of the file after the following commands are entered?<br>
+\`w\`<br>
+\`a\`<br>
+\`NEW\`
+
+1. [x] text text tNEWext
+1. [ ] text text NEWtext
+	> Look above to recall what the \`w\` and \`a\` shortcuts do
+1. [ ] text text textNEW
+	> Look above to recall what the \`w\` and \`a\` shortcuts do
+1. [ ] text textNEW text
+	> Look above to recall what the \`w\` and \`a\` shortcuts do
+1. [ ] text texNEWt text
+	> Look above to recall what the \`w\` and \`a\` shortcuts do
+
+# A file has the following contents:<br>
+\`text\`<br>
+Vim is in normal mode<br><br>
+What are the contents of the file after the following commands are entered?<br>
+\`yy\`<br>
+\`p\`<br>
+\`p\`
+
+1. [x] text<br>text<br>text
+1. [ ] text text text
+	> Look above to recall what the \`yy\` and \`p\` shortcuts do
+1. [ ] texttexttext
+	> Look above to recall what the \`yy\` and \`p\` shortcuts do
+
+# While in normal mode, which of the following commands will yield different results depending on where in the line your cursor is at?
+
+- [x] a
+- [x] w
+- [x] e
+- [x] b
+- [ ] $
+	> This will jump your cursor to the end of the line no matter where it's at
+- [ ] 0
+	> This will jump your cursor to the beginning of the line no matter where it's at
+- [ ] o
+	> This will insert a new line below the current line no matter where the cursor is at
+- [ ] O
+	> This will insert a new line above the current line no matter where the cursor is at
+- [ ] A
+	> This will jump your cursor to the end of the line (and enter insert mode) no matter where it's at
+- [ ] yy
+	> This will copy the current line no matter where your cursor is at
+`;
```

Comment:
Good questions! Might I suggest a couple more that I find really helpful when navigating a file


Certainly! Vim is a powerful text editor, and knowing its basic commands, especially for movement, searching, and find/replace, is crucial. Below are a few questions focused on these aspects of Vim, each designed to test understanding of basic yet essential Vim commands.


## Which of the following commands in Vim moves the cursor to the beginning of the file?

 [x] gg
 [] G - Move cursor to the first character of the last line of the file
 [] ^ - Moves to the first non-blank character of the current line
 [] $ - Moves to the end of the current line



## In Vim, how do you search for the word 'error' in the text?

[] :error
[x] /error
[] ?error
[] *error

Reply:
I like it, I'll add these questions and add the new commands earlier in the section

Reply:
Added 👍